ci/platforms/config.hera

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/bash
 
-export GFS_CI_ROOT=/scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT
+export GFS_CI_ROOT=/scratch1/NCEPDEV/global/glopara/GFS_CI_CD
 export ICSDIR_ROOT=/scratch1/NCEPDEV/global/glopara/data/ICSDIR
 
+# Jenkins directories
+export JENKINS_AGENT_LANUCH_DIR=${GFS_CI_ROOT}/Jenkins/agent
```

Comment:
Just a note that `LAUNCH` is spelled `LANUCH`.

Reply:
duh, thanks.